src/main/scala/io/shiftleft/js2cpg/preprocessing/TranspilationRunner.scala

```diff
@@ -153,16 +149,16 @@ class TranspilationRunner(projectPath: Path, tmpTranspileDir: Path, config: Conf
       // run the transpilers
       workUnit()
 
-      // remove freshly created lock files from transpiler runs
-      PackageJsonParser.LOCKFILES.map(File(projectPath, _)).foreach(_.delete(swallowIOExceptions = true))
+      // remove freshly created files from transpiler runs
+      PackageJsonParser.PROJECT_CONFIG_FILES.map(File(projectPath, _)).foreach(_.delete(swallowIOExceptions = true))
 
       // restore the original package.json
       packageJson.writeText(originalContent)
 
-      // restore lock files
-      PackageJsonParser.LOCKFILES.map(f => File(projectPath, f + ".bak")).collect {
-        case lockFile if lockFile.exists =>
-          lockFile.renameTo(lockFile.pathAsString.stripSuffix(".bak"))
+      // restore config files
+      PackageJsonParser.PROJECT_CONFIG_FILES.map(f => File(projectPath, f + ".bak")).collect {
+        case file if file.exists =>
+          file.renameTo(file.pathAsString.stripSuffix(".bak"))
```

Comment:
I'm not a fan of using `collect/map/...` for side-effects only

```suggestion
      PackageJsonParser.PROJECT_CONFIG_FILES.map(f => File(projectPath, f + ".bak"))
        .filter(_.exists)
        .foreach(file => file.renameTo(file.pathAsString.stripSuffix(".bak"))
```